integration_test.sh

```diff
@@ -1,21 +1,35 @@
 #!/bin/bash
-
 # Build the binary
 make clean
 make
 
 # Start the server
 echo "Running an integration test on the server"
-echo "server { listen 8080; }" >> test_config
+echo "server { listen 8080;
+path /echo EchoHandler;
+path /static1 StaticFileHandler { root www; }
+path /static2 StaticFileHandler { root pics; } }" >> test_config
 ./webserver test_config &>/dev/null &
 
 # Send a request to the server using curl
 echo "Sending a curl request"
-curl -i -s localhost:8080 > test_output
+curl -i -s localhost:8080/echo > test_output_echo
+curl -s localhost:8080/static1/default.html > test_output_static1
+curl -s localhost:8080/static2/ducks.jpg > test_output_static2_jpg
+curl -s localhost:8080/static2/giphy.gif > test_output_static2_gif
+curl -s localhost:8080/static2/mario.jpeg > test_output_static2_jpeg
 
```

Comment:
Nice, good that you tested all files. We need to make sure to update this with future files.